### ISSUE TITLE
Mass Downloader

### ISSUE THREAD

**Issue by melvazmelvazeoll19** [Vencord/plugin-requests#696]
### Plugin Description

I want a plugin where I can upload all the attached files to a message with a single button, for example, you can add a button to the shift menu that will do this.

In the plugin settings, you can add the ability to specify the installation path of the attached files.

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Inbestigator** [comment]
Upload or download?
Either way it seems kinda API spammy

**Comment by melvazmelvazeoll19** [comment]
> Upload or download? Either way it seems kinda API spammy

Download

**Comment by HAHALOSAH** [comment]
> Upload or download? Either way it seems kinda API spammy

Nah, you're downloading from the CDN link, which you don't need authorization to access

**Comment by Vendicated** [comment]
if you're viewing it in chat, you already loaded it anyway, so there's nothing spammy about downloading these files, it's the same amount of requests (if any are even sent at all, the browser likely caches them)
